src/main/src/main/java/org/geoserver/catalog/CatalogBuilder.java

```diff
@@ -1541,10 +1542,11 @@ public LayerInfo buildLayer(CoverageInfo coverage) throws IOException {
      */
     public LayerInfo buildLayer(WMSLayerInfo wms) throws IOException {
         LayerInfo layer = buildLayer((ResourceInfo) wms);
-
         layer.setDefaultStyle(getDefaultStyle(wms));
+        // geos-9312
```

Comment:
Useless comment, tickets go way, I would rather add a comment describing what we are doing if you feel it necessary.